File: backend/src/main/java/com/runway/run/dto/PersonalRecordsResponse.java
package com.runway.run.dto;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class PersonalRecordsResponse {

    private PersonalRecordItemResponse longestRun;
    private PersonalRecordItemResponse fastestPace;
    private PersonalRecordItemResponse mostCalories;
    private PersonalRecordItemResponse best5k;
    private PersonalRecordItemResponse best10k;
    private long totalCompletedRuns;
    private double totalDistanceMeters;
}

File: android/app/src/main/java/com/runway/android/data/running/model/PersonalRecordsResponse.kt
package com.runway.android.data.running.model

data class PersonalRecordsResponse(
    val longestRun: PersonalRecordItem?,
    val fastestPace: PersonalRecordItem?,
    val mostCalories: PersonalRecordItem?,
    val best5k: PersonalRecordItem?,
    val best10k: PersonalRecordItem?,
    val totalCompletedRuns: Long,
    val totalDistanceMeters: Double,
)
